UIforETW/Utility.cpp

```diff
@@ -298,6 +298,47 @@ std::wstring ConvertToCRLF(const std::wstring& input)
 	return result;
 }
 
+std::wstring ReadRegistryString(HKEY root, const std::wstring& subkey, const std::wstring& valueName, bool force32Bit)
+{
+	std::wstring value;
+	const DWORD flags = RRF_RT_REG_SZ | RRF_RT_REG_EXPAND_SZ | RRF_ZEROONFAILURE;

```

Comment:
RegGetValue doesn't like RRF_RT_REG_EXPAND_SZ w/o RRF_NO_EXPAND.
I guess RRF_NO_EXPAND is of little value, hence just remove RRF_RT_REG_EXPAND_SZ?